lib/seafoam/commands.rb

```diff
@@ -186,15 +195,15 @@ def cfg2asm(*args)
     private
 
     # seafoam file.bgv info
-    def info(name, *args)
+    def info(name, formatter_module, *_args)
       file, *rest = parse_name(name)
       raise ArgumentError, 'info only works with a file' unless rest == [nil, nil, nil]
 
-      raise ArgumentError, 'info does not take arguments' unless args.empty?
```

Comment:
I don't really recall. I think I changed it because the command processor never passes arguments down. But, I'll switch it back and potentially revisit when reworking argument processing.